monGARS/core/ray_llm_deployment.py

```
@@ -0,0 +1,58 @@
+        "target_num_ongoing_requests_per_replica": 5,
+    },
+)
+class RayLLMDeployment:
+    def __init__(self):
+        self.llm = LLMIntegration.instance()
+        self.health_check_timestamp = time.time()
+
+    async def health_check(self):
+        """Verify model is responsive"""
+        if time.time() - self.health_check_timestamp > 300:  # 5 minutes
+            test_response = await self.__call__(
```

Ruff (0.14.4)

Local variable `test_response` is assigned to but never used

Remove assignment to unused variable `test_response`

(F841)